src/patternfly/components/Page/page.scss

```
@@ -82,24 +82,17 @@
   }
 
   // Main section
-  --pf-c-page__main-section--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--md--PaddingTop: var(--pf-global--spacer--xl);
-  --pf-c-page__main-section--md--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-page__main-section--md--PaddingBottom: var(--pf-global--spacer--xl);
-  --pf-c-page__main-section--md--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__main-section--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-page__main-section--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-page__main-section--PaddingBottom: var(--pf-global--spacer--2xl);
+  --pf-c-page__main-section--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__main-section--m-table--PaddingTop: calc(var(--pf-global--spacer--sm) / 2);
```

any reason you wouldn't do this?

```suggestion
  --pf-c-page__main-section--m-table--PaddingTop: var(--pf-global--spacer--xs);
```

---

my bad this is meant to be 2px